src/peergos/shared/user/fs/FileTreeNode.java

```diff
@@ -27,7 +29,7 @@
 public class FileTreeNode {
 
     final static int THUMBNAIL_SIZE = 100;
-
+    private enum FileType { Image, Video, Unknown };
```

Comment:
isn't this covered by the mimetype?

Reply:
not quite as there are many different image/video mimetypes. Using a generic enum rather then using mimeType.startsWith() everywhere seemed like a win.

Reply:
But then adding new thumbnails e.g. text, pdf, will all be breaking changes.